core/field_textinput.js

```diff
@@ -156,7 +165,7 @@ Blockly.FieldTextInput.prototype.showEditor_ = function(opt_quietInput) {
       (Blockly.FieldTextInput.FONTSIZE * this.workspace_.scale) + 'pt';
   div.style.fontSize = fontSize;
   htmlInput.style.fontSize = fontSize;
-  /** @type {!HTMLInputElement} */
+
   Blockly.FieldTextInput.htmlInput_ = htmlInput;
```

Comment:
No, because there's only ever one in existence.  Also because we're using it once in the angle picker code, though we probably shouldn't be.